docs/content/fusd/index.md

```diff
@@ -6,43 +6,53 @@ description: FUSD is a 1:1 USD-backed stablecoin on Flow
 
 Flow USD (FUSD) is a stablecoin on Flow, 
```

Comment:
Remove the "on Flow" for smoother reading. This is also redundant as details are provided below.